packages/services/api/src/modules/usage-estimation/resolvers/UsageEstimation.ts

```diff
@@ -0,0 +1,5 @@
+import type { UsageEstimationResolvers } from './../../../__generated__/types.next';
+
+export const UsageEstimation: UsageEstimationResolvers = {
+  /* Implement UsageEstimation resolver logic here */
+};
```

Comment:
That's true! It's common feedback I've been getting.
I'm working on a mode to avoid generating empty object type files like this. Will release that as a feature first before starting the migration 🙂

Reply:
I've made an alpha release to avoid these empty files [in this commit](https://github.com/kamilkisiela/graphql-hive/pull/4740/commits/48c60d5e72423c163011eab48d16f089d00096de).

Now, only a small number of files are created:
- query 
- mutation 
- subscriptions 
- object types with mappers

If an object type _without_ a mapper needs to be created to have field resolvers, it can be created at the expected location on filesystem and it'd be prefilled and automatically wired up to `resolvers.generated.ts`:

https://github.com/kamilkisiela/graphql-hive/assets/33769523/f62e3b9a-fc23-4d53-aed3-ef3688aa4e8c